src/backend/util/register-lti-platforms.ts

```diff
@@ -0,0 +1,148 @@
+import { Provider as ltijs } from 'ltijs'
+import { edusharingMockClientId } from '../../edusharing-mock/server'
+import { edusharingAsToolConfigs } from '../edusharing/get-edusharing-as-tool-configuration'
+import { serverLog } from '../../utils/server-log'
+import config from '../../utils/config'
+
+export async function registerLtiPlatforms() {
+  if (config.ENVIRONMENT === 'staging') {
+    await registerSaltire()
+
+    // Register platform: itslearning
+    await registerPlatform({
+      url: config.ITSLEARNING_URL,
+      name: config.ITSLEARNING_NAME,
+      clientId: config.SERLO_EDITOR_CLIENT_ID_ON_ITSLEARNING,
+      authenticationEndpoint: config.ITSLEARNING_AUTHENTICATION_ENDPOINT,
+      accesstokenEndpoint: config.ITSLEARNING_ACCESS_TOKEN_ENDPOINT,
+      key: config.ITSLEARNING_KEYSET_ENDPOINT,
+    })
+
+    // Register platform: edu-sharing (RLP)
+    const platform = await registerPlatform({
+      url: config.EDUSHARING_RLP_URL,
+      name: config.EDUSHARING_RLP_NAME,
+      clientId: config.SERLO_EDITOR_CLIENT_ID_ON_EDUSHARING_RLP,
+      authenticationEndpoint: config.EDUSHARING_RLP_AUTHENTICATION_ENDPOINT,
+      accesstokenEndpoint: config.EDUSHARING_RLP_ACCESS_TOKEN_ENDPOINT,
+      key: config.EDUSHARING_RLP_KEYSET_ENDPOINT,
+    })
+    if (platform) {
+      edusharingAsToolConfigs.push({
+        issWhenEdusharingLaunchedSerloEditor: config.EDUSHARING_RLP_URL,
+        loginEndpoint: config.EDUSHARING_RLP_LOGIN_ENDPOINT,
+        launchEndpoint: config.EDUSHARING_RLP_LAUNCH_ENDPOINT,
+        clientId: config.EDUSHARING_RLP_CLIENT_ID_ON_SERLO_EDITOR,
+        detailsEndpoint: config.EDUSHARING_RLP_DETAILS_ENDPOINT,
+        keysetEndpoint: config.EDUSHARING_RLP_KEYSET_ENDPOINT,
+      })
+      serverLog('Registered tool: edu-sharing (RLP)')
+    }
+  }
+
+  if (config.ENVIRONMENT === 'local') {
+    await registerSaltire()
+
+    // Register platform: edusharing mock
+    const platform = await registerPlatform({
```

Comment:
Super tiny, but since you did refactor this file, I would also change the name of this constant to make it more specific:
```suggestion
    const edusharingPlatform = await registerPlatform({
```